.github/workflows/release-maven-publish.yml

```diff
@@ -19,6 +19,15 @@ jobs:
         distribution: 'zulu'
         java-version: 8
 
+    - name: Relase
```

Comment:
is this a typo? likely doesnt matter but thought i should mention. @Rodhlann

Reply:
It definitely is! Thanks for the heads up